### PR TITLE
web design

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -10,57 +10,71 @@
...
+    <p><a href="https://github.com/denolib/awesome-deno">Other Deno resources.</a>
 
-    <h2>Getting started</h2>
+    <h2>Install</h2>
 
-    <p>Install Deno into ~/.deno/bin
-<h2>With Shell</h2>
+<p>With Shell
 <pre>
 curl -L https://deno.land/x/install/install.sh | sh
```
Comment (hayd) [inline website/index.html]: Aside: perhaps this should be -fL

File: Docs.md [hunk 2]
```diff
@@ -63,6 +63,92 @@ Or see the [doc website](https://deno.land/typedoc/index.html).
...
+### Prerequisites:
+
+To ensure reproducible builds, deno has most of its dependencies in a git
+submodule. However, you need to install separately:
+
+1. [Rust](https://www.rust-lang.org/en-US/install.html) >= 1.31.1
+2. [Node](https://nodejs.org/)
+3. Python 2.
+   [Not 3](https://github.com/denoland/deno/issues/464#issuecomment-411795578).
+4. [ccache](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/ccache)
```
Comment (piscisaureus) [inline Docs.md]: ccache is no longer necessary and I don't think we're using it ourselves.
Ergo, this line should be dropped.

File: Docs.md [hunk 3]
```diff
@@ -63,6 +63,92 @@ Or see the [doc website](https://deno.land/typedoc/index.html).
 If you are embedding deno in a Rust program, see
 [the rust docs](https://deno.land/rustdoc/deno/index.html).
 
+## Build Instructions
+
+Follow these steps if you are trying to submit a PR.
```
Comment (piscisaureus) [inline Docs.md]: This line seems lost within it's surrounding context.

File: Docs.md [hunk 4]
```diff
@@ -63,6 +63,92 @@ Or see the [doc website](https://deno.land/typedoc/index.html).
...
+
+### Build:
+
+    # Fetch deps.
+    git clone --recurse-submodules https://github.com/denoland/deno.git
+    cd deno
+    ./tools/setup.py
+
+    # You may need to ensure that sccache is running.
+    # (TODO it's unclear if this is necessary or not.)
```
Comment (piscisaureus) [inline Docs.md]: Not necessary.

File: website/index.html [hunk 5]
```diff
@@ -10,57 +10,71 @@
...
-<h2>With PowerShell</h2>
+<p>With PowerShell
 <pre>
 iex (iwr https://deno.land/x/install/install.ps1)
 </pre>
 
-Try a Deno program. Install by bash alias.
-This one serves a local directory in HTTP.
+    <h2>Mini-tutorial</h2>
+
```
Comment (piscisaureus) [inline website/index.html]: Messy markup. Would suggest running `prettier --write index.html`.

Comment (ry) [reply website/index.html]: Done.

File: Docs.md (hunk 4 above)
Comment (ry) [reply Docs.md]: This was added recently by someone claiming that they needed this to get going. I also think it's unnecessary - but let's leave it a bit longer.

File: Docs.md (hunk 3 above)
Comment (ry) [reply Docs.md]: Removed.

File: Docs.md (hunk 2 above)
Comment (ry) [reply Docs.md]: Removed.

File: website/index.html (hunk 1 above)
Comment (ry) [reply website/index.html]: Fixed.

**Pull request merged**